packages/create-cloudflare/src/helpers/command.ts

```diff
@@ -18,12 +18,14 @@ type Command = string | string[];
 
 type RunOptions = {
 	startText?: string;
-	doneText?: string;
+	doneText?: string | ((result: string) => string);
```

Comment:
In line with my other recommendation, the param here would be better named `output`

```suggestion
	doneText?: string | ((output: string) => string);
```

It would be clearer to understand what their purpose is without having to read the source code